src/ILAMB/ccgfilt.py

```diff
@@ -0,0 +1,1057 @@
+
```

Comment:
Where did this come from? Did you code it up or get the file from somewhere? I am just wondering if we should include it in the ILAMB codebase or rather depend on it if elsewhere. Note sure.